docs/04-operation-guides/operations/10-backup-kyma.md

```diff
@@ -27,8 +27,7 @@ You can create on-demand volume snapshots manually, or set up a periodic job tha
 
 ## Back Up Resources Using Velero
 
-You can back up and restore individual resources manually or automatically with Velero. For more information, read the [Velero documentation](https://velero.io/docs/).
-Be aware that a full backup of a Kyma cluster isn't supported. Start with the existing Kyma installation and restore specific resources individually.
+Velero backup is not supported at this time.
```

Comment:
You can back up and restore individual resources manually or automatically using third-party tools like Velero. For more information, read the [Velero documentation](https://velero.io/docs/).

Please be aware that these tools may have limitations and might not support fully automated cluster backups.

Additionally, they often require specific access rights to your infrastructure provider, which may not be available in Kyma's managed offerings where you lack sufficient access rights to the infrastructure account.